docs/building-a-form/about-the-json-schema-standard-and-react-jsonschema-form.md

```diff
@@ -0,0 +1,298 @@
+  - [Field components](#field-components)
+  - [About the `onChange` field component](#about-the-onchange-field-component)
+  - [About the `uiSchema` field component](#about-the-uischema-field-component)
+- [How schemaform uses rjsf](#how-schemaform-uses-rjsf)
+  - [Customizing fields and widgets from rjsf](customizing-fields-and-widgets-from-rjsf)
+  - [Supporting multi-page forms](#supporting-multi-page-forms)
+
+### Understanding JSON Schema
+
+The JSON Schema standard describes the allowed shape of JSON objects. For more information about the JSON Schema Standard, see [Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/).
+
+You can nest schemas as far down as you'd like. Schemas have a type, which tells you what kind of data is allowed:
+
+```
+{
+  type: 'string'
+}
+```
+
+The JSON schema can also have validation information, such as regexes or length requirements. This example allows any string that's at least two characters and only contains `e` and `f`, meaning `eff` is valid, but `fcc` is not:
+
+```
+{
+  type: 'string',
+  pattern: '^[ef]*$',
+  minLength: 2
+}
+```
+
+You can also specify some built-in `format` values for strings, such as `email`, as a shortcut for including your own patterns.
+
+##### Describing object fields and arrays
+
+This example describes a JSON document that is an object with one property called `myField`, which is a number, meaning `{ myField: 2 }` would be valid:
+
+```
+{
+  type: 'object',
+  properties: {
+    myField: {
+      type: 'number'
+    }
+  }
+}
+```
+
+However, `{}` is also valid. To require a property in an object, use the `required` property. As in this example, `required` is always on the object that contains the field, not the field itself:
+
+```
+{
+  type: 'object',
+  required: ['myField'],
+  properties: {
+    myField: {
+      type: 'number'
+    }
+  }
+}
+```
+
+Arrays work similarly to objects. This example describes an array of boolean values: `[true, false, true]`. Items can be an object schema or any other type of schema as well:
+
+```
+{
+  type: 'array',
+  items: {
+    type: 'boolean'
+  }
+}
+```
+
+Many libraries implement the JSON Schema specification and let you validate that an object matches a given schema. The US Forms System uses [ajv](https://www.npmjs.com/package/ajv) for unit tests and [jsonschema](https://www.npmjs.com/package/jsonschema) in the schemaform code. For more examples of JSON Schema projects, see the [JSON Schema site example list](http://json-schema.org/examples.html).
+
+### Understanding react-jsonschema-form (rjsf)
+
+[react-jsonschema-form](https://github.com/mozilla-services/react-jsonschema-form), or *rjsf*, generates a form from a JSON Schema, in addition to other UI information. To generate a form, react-jsonschema-form steps through the schema depth and renders different React components based on the type of data each property in the schema represents.
+
+##### About the `Form` component
```

Comment:
This header is a bit confusing. We're explaining a lot more than just what the `Form component` is here; we're also defining `fields` and `widgets`, which are important concepts that I think deserve their own sections.

Reply:
I added "fields and widgets" to the header here for now, but this is another example of really needing to organize the content in this topic (https://github.com/usds/us-forms-system/issues/139).